### PR TITLE
Add PyCIEMSS context

### PR DESCRIPTION
**Pull request by fivegrant**
On context start, the PyCIEMSS `optimize` function is provided alongside a loaded model.

Feel free to try with:
```
{
	"model_config_id": "04980719-6be1-46fc-bea3-0c0653b056c0",
	"optimize": {
		"risk_bound": 200,		
		"end_time": 20.0,
                ...etc (look in the optimize procedures to see all options)
	}
}
```

# Outstanding issues
- [x] Delete cell above `optimize`. I realize we might not want to do this since the user may want to run some code before optimize.
- [x] Populate default payload. `default_payload.json` is correct but it's not rendering in the Dev UI for some reason.

### CODE REVIEW

**Comment by fivegrant** [comment]
The `default_payload.json` always worked as I suspected. Just needed to rebuild.

File: src/askem_beaker/contexts/pyciemss/procedures/python3/optimize.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Define needed variables in above cell(s)
+model # Selected model is loaded as a dictionary 
+
+result = optimize(
+    model_path_or_json = model,
+    static_parameter_interventions = param_value_objective([], [torch.tensor({{ start_time | default("0.0")}})]),
+    objfun = lambda x: np.sum(np.abs(x)),
+    initial_guess_interventions =  [],
```
Comment (brandomr) [inline src/askem_beaker/contexts/pyciemss/procedures/python3/optimize.py:8]: should this be set as part of the payload?

File: src/askem_beaker/contexts/pyciemss/procedures/python3/optimize.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    bounds_interventions = [],
```
Comment (brandomr) [inline src/askem_beaker/contexts/pyciemss/procedures/python3/optimize.py:9]: same as above, should this be part of the setup payload?

**Review by Tom-Szendrey — APPROVED** [review]
Thank you for this!

**Pull request merged**